x-pack/auditbeat/module/system/user/_meta/fields.yml

```diff
@@ -0,0 +1,56 @@
+- name: user
+  type: group
+  description: >
+    `user` contains information about the users on a system.
+  release: experimental
+  fields:
+    - name: name
+      type: keyword
+      description: >
+          User name.
+    - name: uid
+      type: integer
+      description: >
+          User ID.
+    - name: gid
+      type: integer
+      description: >
+          Group ID.
+    - name: dir
+      type: keyword
+      description: >
+          User's home directory.
+    - name: shell
+      type: keyword
+      description: >
+          Program to run at login.
+    - name: user_information
+      type: text
+      description: >
+          General user information. On Linux, this is the gecos field.
+    - name: group
+      type: group
+      description: >
+        `group` contains information about any groups the user is part of (beyond the user's primary group).
+      fields:
+        - name: name
+          type: keyword
+          description: >
+              Group name.
+        - name: gid
+          type: integer
+          description: >
+              Group ID.
+    - name: password
+      type: group
+      description: >
+        `password` contains information about a user's password (not the password itself).
```

Comment:
Can you also document what the possible values are for this enum.

Reply:
Done.